test/list/main.go

```diff
@@ -262,12 +328,34 @@ func main() {
 			log.Fatalf("Error walking: %v", err)
 		}
 	}
+
 	if *dumpJson {
 		json, err := json.Marshal(tests.tests)
 		if err != nil {
 			log.Fatal(err)
 		}
 		fmt.Println(string(json))
+	} else if *writeConformanceDoc {
+		file, err := os.Create("Conformance.md")
```

Comment:
use `conformance.md` which is the dominant convention in Kubernetes.

Reply:
Fixed.